src/main/java/cn/nukkit/entity/passive/EntityChicken.java

```diff
@@ -38,7 +38,7 @@ public String getName() {
 
     @Override
     public Item[] getDrops() {
-        return new Item[]{Item.get(Item.RAW_CHICKEN), Item.get(Item.FEATHER)};
+        return new Item[]{Item.get(((this.isOnFire()==true) ? Item.COOKED_CHICKEN : Item.RAW_CHICKEN)), Item.get(Item.FEATHER)};
```

Comment:
Why `this.isOnFire()==true` and not just `this.isOnFire()`?

Reply:
My kind of formatting I like, sorry if this is distrubing you can change it if you want

Reply:
there is. It makes the code less readable